### PR TITLE
Throwing.new items.july2023

### PR DESCRIPTION
**Pull request by Diggles27**

### CODE REVIEW

**Review by namidaka — CHANGES_REQUESTED** [review]
Shuriken should have a low tier version a mid tier version and a high tier one.

**Comment by Diggles27** [comment]
made throwing knives/daggers consistent. Knife = cut, Dagger = pierce

File: items/weapons.xml [hunk 1]
```diff
@@ -1840,63 +1840,63 @@
...
+  <CraftedItem id="crpg_highland_throwing_dagger_h1" name="{Diggles}Steel Throwing Knife +1" crafting_template="crpg_ThrowingKnife" culture="Culture.battania" modifier_group="knife_throwing">
     <Pieces>
       <Piece id="crpg_highland_throwing_dagger_blade_h1" Type="Blade" scale_factor="90" />
       <Piece id="crpg_highland_throwing_dagger_guard_h1" Type="Guard" />
       <Piece id="crpg_highland_throwing_dagger_handle_h1" Type="Handle" />
       <Piece id="crpg_highland_throwing_dagger_pommel_h1" Type="Pommel" />
     </Pieces>
   </CraftedItem>
-  <CraftedItem id="crpg_highland_throwing_dagger_v1_h2" name="{=koOrDa9q}Highland Throwing Dagger +2" crafting_template="crpg_ThrowingKnife" culture="Culture.battania" modifier_group="knife_throwing">
+  <CraftedItem id="crpg_highland_throwing_dagger_h2" name="{Diggles}Steel Throwing Knife +2" crafting_template="crpg_ThrowingKnife" culture="Culture.battania" modifier_group="knife_throwing">
```
Comment (namidaka) [inline items/weapons.xml:1891]: v2

**Review by namidaka — CHANGES_REQUESTED** [review]

File: items/weapons.xml (hunk 1 above)
Comment (Diggles27) [reply items/weapons.xml]: fix submitted

**Review by namidaka — APPROVED** [review]

**Pull request merged**